src/Controls/src/Core/Platform/GestureManager/GesturePlatformManager.Android.cs

```diff
@@ -150,7 +150,9 @@ ScaleGestureDetector InitializeScaleDetector()
 			var context = Control.Context;
 			var listener = new InnerScaleListener(new PinchGestureHandler(() => View), context.FromPixels);
 			var detector = new ScaleGestureDetector(context, listener, Control.Handler);
+#pragma warning disable CS0618 // Type or member is obsolete
 			ScaleGestureDetectorCompat.SetQuickScaleEnabled(detector, true);
```

Comment:
And the new here?

Reply:
I was able to find replacement

https://developer.android.com/reference/kotlin/androidx/core/view/ScaleGestureDetectorCompat#setQuickScaleEnabled(java.lang.Object,boolean)